srcpkgs/php/template

```diff
@@ -23,7 +23,14 @@ replaces="php-mcrypt<7.2.0"
 if [ -n "$CROSS_BUILD" ]; then
 	# php-pear needs php to build
 	hostmakedepends+=" php"
-	CFLAGS+=" -DHAVE_LIBDL -DHAVE_DLOPEN -DHAVE_DLSYM"
+	CFLAGS+=" -DHAVE_LIBDL \
+		  -DHAVE_DLOPEN \
+		  -DHAVE_DLSYM
+		  -DHAVE_SHM_IPC \
+		  -DHAVE_SHM_MMAP_ANON \
+		  -DHAVE_SHM_MMAP_ZERO \
+		  -DHAVE_SHM_MMAP_POSIX \
+		  -DHAVE_SHM_MMAP_FILE"
```

Comment:
I think this should work without `\`

And consider using one tab + one space for indenting the string.

Reply:
Done